fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/explain/ExplainLevel.java

```diff
@@ -20,25 +20,41 @@
 
 package com.apple.foundationdb.record.query.plan.explain;
 
+import java.util.Objects;
+
 /**
  * Explain level. Adjusts the granularity of explain output.
  */
-public final class ExplainLevel {
+public enum ExplainLevel {
```

Comment:
Let's not introduce yet another naming of the same underlying thing. Can't you make the ExplainLevel an enum directly?